### PR TITLE
Include middleware type definition when creating releases

### PR DESCRIPTION
**Pull request by nnyegaard**
Included the new middleware.d.ts type definition file in the package.json files array.

### CODE REVIEW

**Comment by chanlito** [comment]
It'd be nice to have this, I currently using my own type def file

File: package.json [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
   "main": "./index.js",
-  "files": [
-    "src",
-    "typescript",
-    "index.js",
-    "index.d.ts",
-    "middlewares.js",
-    "!__tests__"
-  ],
+  "files": ["src", "index.js", "index.d.ts", "middlewares.js", "middlewares.d.ts", "!__tests__"],
```
Comment (lmammino) [inline package.json]: Any reason to change this?

File: package.json [hunk 2]
```diff
@@ -28,12 +21,7 @@
     "type": "git",
     "url": "git+https://github.com/middyjs/middy.git"
   },
-  "keywords": [
-    "Lambda",
-    "Middleware",
-    "Serverless",
-    "Framework"
-  ],
+  "keywords": ["Lambda", "Middleware", "Serverless", "Framework"],
```
Comment (lmammino) [inline package.json]: also this

File: package.json [hunk 3]
```diff
@@ -70,8 +58,6 @@
     "querystring": "^0.2.0"
   },
   "jest": {
-    "collectCoverageFrom": [
-      "src/**/*.js"
-    ]
+    "collectCoverageFrom": ["src/**/*.js"]
```
Comment (lmammino) [inline package.json]: I suppose you have some automatic formatter at this stage... :D

**Comment by nnyegaard** [comment]
Yeah. I will look into adding an .editorconfig file in a new PR, so that
the formatting is correct ( If that is possible). If there is interest for
that?

Den lør. 24. mar. 2018 11.58 skrev Luciano Mammino <notifications@github.com

**Pull request merged**